src/domain/books/BooksPage.tsx

```diff
@@ -1,7 +0,0 @@
-export const BooksPage = (): JSX.Element => {
-  return (
-    <div>
-      Books Page
-    </div>
-  )
-}
```

Comment:
You have to follow the folder structure of the app. Please don't create things by yourself.

Reply:
Sure Sir I will keep it in mind